### PR TITLE
reenable enableSwaggerCodegen at gradle sample

### PR DESCRIPTION
**Pull request by mshima**
Related to https://github.com/jhipster/generator-jhipster/issues/22586.

Switch to 7.0.0-beta which fixes the bug.

---

Please make sure the below checklist is followed for Pull Requests.

- [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
- [ ] Tests are added where necessary
- [ ] The JDL part is updated if necessary
- [ ] [jhipster-online](https://github.com/jhipster/jhipster-online) is updated if necessary
- [ ] Documentation is added/updated where necessary
- [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/main/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (below reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

**Review by DanielFran — APPROVED** [review]

File: generators/server/templates/gradle/swagger.gradle.ejs [hunk 1]
```diff
@@ -36,6 +36,15 @@ openApiGenerate {
     importMappings = [Problem:"org.springframework.http.ProblemDetail"]
 }
 
+configurations {
+    all {
+        resolutionStrategy {
+            // TODO drop forced version. Refer to https://github.com/OpenAPITools/openapi-generator/issues/15876
+            force "io.swagger.parser.v3:swagger-parser:2.1.15"
```
Comment (DanielFran) [inline generators/server/templates/gradle/swagger.gradle.ejs:43]: https://stackoverflow.com/questions/76288749/gradle-built-of-micronaut-based-app-breaks-with-nosuchmethoderror-from-3rd-party

Maybe we need to use 1.33?

Comment (mshima) [reply generators/server/templates/gradle/swagger.gradle.ejs]: snakeyaml 2.0 is a major version.
Probably incompatible.
At link you provided says it’s not enough.

v2 is dependency of jackson 2.15.x

**Comment by mshima** [comment]
I’ve tested locally and resolutionStrategy is not working for the use case. Dependency is not been forced.

**Comment by mraible** [comment]
Any updates on this one @mshima?

**Comment by mshima** [comment]
No approach I’ve found in the web forces the dependency to the desired one.

**Comment by atomfrede** [comment]
I didn't follow sorry. Do we need to force a certain version and it's not working?

**Comment by mshima** [comment]
@atomfrede yes, there is a dependency conflict that makes open api generator to fail.

**Comment by atomfrede** [comment]
I have some time next week and try to have look.

**Comment by mshima** [comment]
@DanielFran what do you think?
Should we wait for final?

**Review by DanielFran — APPROVED** [review]

**Pull request merged**